backend/backend/Services/Contracts/IQuotationService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using backend.Models.Response;
+using backend.Models.General;
+using backend.DataAccess.Database.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace backend.Services.Contracts
+{
+    public interface IQuotationService
+    {
+       public bool NewQuotation(QuotationModel model);
+       public QuotationResponseModel GetById(int id);
+       public List<QuotationResponseModel> GetAll();
+       public void Delete(int id);
+       public QuotationResponseModel Update(QuotationModel model);
```

Comment:
No need to put in public for the methods, once the interface is public the implementing class will be able to read the methods if they are default access modifier. Just note for clean code no problem